test/CMakeLists.txt

```diff
@@ -70,12 +70,13 @@ set(stdexec_test_sources
     exec/async_scope/test_spawn_future.cpp
     exec/async_scope/test_empty.cpp
     exec/async_scope/test_stop.cpp
+    tbbexec/test_tbb_thread_pool.cpp
     )
 
 add_executable(test.stdexec ${stdexec_test_sources})
 
 target_include_directories(test.stdexec PRIVATE ${CMAKE_CURRENT_LIST_DIR})
-target_link_libraries(test.stdexec PUBLIC STDEXEC::stdexec stdexec_executable_flags Catch2::Catch2)
+target_link_libraries(test.stdexec PUBLIC STDEXEC::stdexec stdexec_executable_flags Catch2::Catch2 TBB::tbb fmt::fmt)
```

Comment:
How do you want to deal with the tbb dependency? I assume you don't want to depend on it entirely. I'm not a CMake expert, though...

Reply:
I've stared at enough CMake and happy to help here if needed. It's relatively easy to either put it behind an explicit flag (generally my preference) or to only attempt to look for TBB and disable support if it's not found (instead of erroring out).

Reply:
@msimberg lmk what you'd like to see and how to spell it for cmake.

I plan to drop `fmt` before this is merged (I guess it's C++20 so I can just use `<format>` 🤦. I'm still at 17 for day-to-day work for a few more months.)

Reply:
@BenFrantzDale I think you might actually be fine mostly duplicating what's been done for CUDA support: https://github.com/NVIDIA/stdexec/blob/41ec3de5bb9bfb24ac18e3d736f2aaa2fd65d533/CMakeLists.txt#L193-L251. Replacing `CUDA` with with `TBB` and `nvexec` with `tbbexec` of course. However, I don't think you need any of the `target_compile_features`/`target_compile_options` calls. The stuff you want to keep is setting up the target: https://github.com/NVIDIA/stdexec/blob/41ec3de5bb9bfb24ac18e3d736f2aaa2fd65d533/CMakeLists.txt#L205-L208, linking to `stdexec`: https://github.com/NVIDIA/stdexec/blob/41ec3de5bb9bfb24ac18e3d736f2aaa2fd65d533/CMakeLists.txt#L211, and finding and linking to the `tbb` target: https://github.com/NVIDIA/stdexec/blob/41ec3de5bb9bfb24ac18e3d736f2aaa2fd65d533/CMakeLists.txt#L235-L242.

Finally, for the test I'm not sure if @ericniebler would prefer a separate test executable (as for CUDA) or if it's fine to have it in the main executable? If the former you'll want to look at https://github.com/NVIDIA/stdexec/blob/main/test/nvexec/CMakeLists.txt. If the latter you'll need to conditionally include the test file and targets with `$<$<BOOL:STDEXEC_ENABLE_TBB>:tbbexec/test_tbb_thread_pool.cpp>` and `$<$<BOOL:STDEXEC_ENABLE_TBB>:tbbexec>`, respectively.

Let me know if any of this doesn't make sense.

Reply:
> Finally, for the test I'm not sure if @ericniebler would prefer a separate test executable (as for CUDA) or if it's fine to have it in the main executable?

Probably a separate executable would be cleanest. I would actually love if _all_ the tests were in their own executable, but I haven't had time to make that change.